delta-lake/src/integration/java/org/apache/iceberg/delta/TestSnapshotDeltaLakeTable.java

```diff
@@ -65,26 +68,23 @@ public class TestSnapshotDeltaLakeTable extends SparkDeltaLakeSnapshotTestBase {
   private static final String NAMESPACE = "delta_conversion_test";
   private static final String defaultSparkCatalog = "spark_catalog";
   private static final String icebergCatalogName = "iceberg_hive";
-  private String partitionedIdentifier;
-  private String unpartitionedIdentifier;
-  private String externalDataFilesIdentifier;
-  private String typeTestIdentifier;
-  private final String partitionedTableName = "partitioned_table";
-  private final String unpartitionedTableName = "unpartitioned_table";
-  private final String externalDataFilesTableName = "external_data_files_table";
-  private final String typeTestTableName = "type_test_table";
-  private final String snapshotPartitionedTableName = "iceberg_partitioned_table";
-  private final String snapshotUnpartitionedTableName = "iceberg_unpartitioned_table";
-  private final String snapshotExternalDataFilesTableName = "iceberg_external_data_files_table";
-  private final String snapshotNewTableLocationTableName = "iceberg_new_table_location_table";
-  private final String snapshotAdditionalPropertiesTableName =
+  private static final String partitionedTableName = "partitioned_table";
```

Comment:
as best practice for testing, we can just hard code these names to individual tests, no need to make them static variables